features/util.py

```diff
@@ -140,9 +173,13 @@ def build_debs(
             check=True,
         )
 
+    if os.path.exists(SBUILD_DIR):
+        shutil.rmtree(SBUILD_DIR)
+    os.makedirs(SBUILD_DIR)
```

Comment:
I think we can turn this into a try..except block:
```python
try:
  shutil.rmtree(SBUILD_DIR)
except FileNotFoundError:
  os.makedirs(SBUILD_DIR)
  ```

Reply:
I think the current version reads just a little bit better than the `try/except` logic to me imho.

But if I am outvoted, I will change it :)